### ISSUE TITLE
Error: ENOENT: no such file or directory, open '.pnpm/@mui+toolpad@0.1.47_prop-types@15.8.1_webpack@5.90.0/node_modules/@mui/toolpad/dist/editor/index.html'

### ISSUE THREAD

**Issue by slim-hmidi** [mui/mui-toolpad#3129]
### Steps to reproduce

Link to live example: (required)

Steps:
1. I followed the documentation steps 
2. pnpm create toolpad-app my-toolpad-app
3. run the pnpm run dev

### Current behavior

I can not launch the toolpad when I run the dev command.

### Expected behavior

It should launch correctly the toolpad.

### Your environment

<details>
  <summary><code>npx @mui/envinfo</code></summary>

```
  System:
    OS: macOS 14.3
  Binaries:
    Node: 20.10.0 - /usr/local/bin/node
    npm: 10.2.3 - /usr/local/bin/npm
    pnpm: 8.14.3 - ~/Library/pnpm/pnpm
  Browsers:
    Chrome: Not Found
    Edge: Not Found
    Safari: 17.3
  npmPackages:
    @mui/toolpad: ^0.1.47 => 0.1.47 
```
</details>

**Search keywords**: *

**Comment by prakhargupta1** [comment]
Yes, today I got this error when upgrading to toolpad version 47. https://github.com/mui/mui-private/pull/376

**Comment by Janpot** [comment]
Looks like a release went wrong. Releasing again in https://github.com/mui/mui-toolpad/pull/3131

**Comment by Janpot** [comment]
Fixed in v0.1.48. I'm sorry for the inconvienence.
